### PR TITLE
Desktop Version

### PR DESCRIPTION
**Pull request by MoRaad97**
In this PR we:

- Add the hover and active feature for all buttons.
- Upload the images for the desktop version.
- Add the below sections in the responsive code design method: 
1.  "Toolbar" section.
2.  "headline" section.
3.  "recent-work" section.
4.  "about" section.
5.  "contact form" section.
6.  "footer" section.
- Modify the mobile version design to work responsively with the desktop version.

### CODE REVIEW

**Review by youmari — CHANGES_REQUESTED** [review]
Hi @MoRaad97 , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that 

- [x] Doesn't have a Descriptive short summary 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -33,8 +41,9 @@
       </div>
     </header>
     <!-- Toolbar end -->
+
     <!--  headline section start -->
-    <section id="headline">
+    <section class="headline">
       <div class="headline-container container">
         <div class="text">
           <h1>Hey There. I’m Mohammed</h1>
```
Comment (koomekevo) [inline index.html:49]: @MoRaad97 Kindly add spacing between the `Hey there I am Mohammed` sentence and `I am a software developer` sentence as shown in the below screenshot as per the Figma design.

mohaheytherecorrect 

The below screenshot is your design 

mohaheytherewrongedited

File: index.html [hunk 2]
```diff
@@ -82,127 +91,175 @@ <h3>I’am a Software Developer</h3>
...
       </div>
     </section>
     <!--  headline section end -->
+
     <!--  work section start -->
-    <section id="recent-work">
+    <section class="recent-work">
       <div class="container">
         <div class="section-header">
           <h1>My Recent Work</h1>
```
Comment (koomekevo) [inline index.html:107]: @MoRaad97 Kindly modify the `My recent work` section according to the content of the Figma template. Four cards do not resemble the content of the Figma template which I have circled in the below screenshot. 

moharecentworkswrongedited

Below is the correct screenshot showing the correct content of the `My recent works` section

moharecentworkscorrect

**Review by koomekevo — CHANGES_REQUESTED** [review]
Hi @MoRaad97, 



Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
:heavy_check_mark: You have implemented the media queries correctly.
:heavy_check_mark: Your README and Pull Request are descriptive.
:heavy_check_mark: Your project has no linter errors.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by danielufeli — APPROVED** [review]
Hi @MoRaad97, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

a

## Highlights
- Nice commits messages✔️
- Descriptive PR ✔️
- Clean Design ✔️
- Good use of Github flow ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @danielufeli in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**